### ISSUE TITLE
Suppress custom `tsconfig.json` warning 

### ISSUE THREAD

**Issue by Dig-Doug** [bazelbuild/rules_nodejs#808]
Hi,

I would like to be able to suppress this warning:

```
WARNING: your tsconfig.json file specifies options which are overridden by Bazel:
 - compilerOptions.paths is determined by the module_name attribute in transitive deps[]
 - compilerOptions.rootDir and compilerOptions.baseUrl are always the workspace root directory
```

In my `tsconfig.json`, I have changed some of the configurations to work with my IDE, e.g.:

```
    "baseUrl": ".",
    "paths": {
      # Helps IDE find generated files
      "<workspace-name>/*": [
        "*",
        "bazel-bin/*"
      ]
    }
```

I'm aware that bazel will override these options, which is totally fine. However, when I run my build the message above appears 10s-100s of times and makes the build log super long.

It would be great if I could:

- Suppress the warning
- Only print it once
- Somehow specify that the user knows that these options will be override and that's OK

Thanks!

**Comment by Toxicable** [comment]
Looks like this was removed already in https://github.com/bazelbuild/rules_typescript/commit/8d8d398caf7d77620e66502adc51cc1a83def0f6

**Comment by Dig-Doug** [comment]
Cool, thanks!
